app.py

```diff
@@ -152,6 +166,44 @@ def render_login():
     return render_template('login.html')
 
 
+@app.route('/logout', methods=['GET'])
+@login_required
+def logout():
+    """Logout a user."""
+    logout_user()
+    return redirect(url_for('/'))
+
+
+@app.route('/test/register/<username>/<password>')
+def test_register(username, password):
+    """Route for test register."""
+    user = User(username=username, hashed_password=int(password))
+    scoped_session = db.create_scoped_session()
+    res = db_manager.DBManager(scoped_session).add_user(user)
+
+    if res.is_ok():
+        return "you are registered {}".format(user.username), 200
+    return "{}".format(res), 400
```

Comment:
do we need these test endpoints? We should add unit tests that exercise the regular endpoints instead